.vscode/launch.json

```diff
@@ -10,7 +10,10 @@
       "args": [
         "-config",
         "${workspaceRoot}/.config.yaml"
-      ]
+      ],
+      "env": {
+        "PATH": "${workspaceRoot}/bin/.getenvoy/builds/standard/1.17.1/darwin/bin/:${env:PATH}"
+      }
```

Comment:
I don't think we want this?

Reply:
otherwise it picks up random envoy from $PATH
what is your suggestion?

Reply:
This has the OS and version hardcoded; I'm not sure if it will even work well on M1 chips since getenvoy doesn't support arm.

Would prefer something this opinionated isn't in a shared config.  Maybe it just shouldn't be in the repo, or we should make a more consistent location for the envoy binary.  We should tackle that separately.